eng/common/scripts/apiview-helper.ps1

```diff
@@ -0,0 +1,56 @@
+
+
+function Map-LanguageToAPiview-Filter($language)
+{
+    $lang = $language
+    # Update language name to match those in API cosmos DB. Cosmos SQL is case sensitive and handling this within the query makes it slow.
+    if($lang -eq 'javascript'){
+        $lang = "JavaScript"
+    }
+    elseif ($lang -eq "dotnet"){
+        $lang = "C#"
+    }
+    elseif ($lang -eq "java"){
+        $lang = "Java"
+    }
+    elseif ($lang -eq "python"){
+        $lang = "Python"
+    }
+    return $lang
+}
+
+function Check-ApiReviewStatus($packageName, $language)
+{
+  # Get API view URL and API Key to check status
+  Login-AzAccount
+  $url = Get-AzKeyvaultSecret -VaultName "AzureSDKPrepRelease-KV" -Name APIURL -AsPlainText
+  $apiKey = Get-AzKeyvaultSecret -VaultName "AzureSDKPrepRelease-KV" -Name APIKEY -AsPlainText
+
+  $lang = Map-LanguageToAPiview-Filter -language $language
+  $headers = @{ "ApiKey" = $apiKey }
+  $body = @{
+    language = $lang
+    packageName = $packageName
```

Comment:
I suspect we will eventually also need to include the package version. Should we go ahead and include it even if we aren't using it yet on the apiview side?

Reply:
done.